exercises/practice/scrabble-score/scrabble_score.go

```diff
@@ -1 +1,5 @@
 package scrabble
+
+func Score(word string) int {
+  panic("implement me")
```

Comment:
```suggestion
  panic("Please implement the Score function")
```

A suggestion to make this comment similar to the other ones we have on the stubs.